app/js/components/homePage/manageApps/AddAddon.jsx

```diff
@@ -14,29 +14,37 @@ export default function AddAddon({ handleClear, handleUpload }) {
     <div className="container-fluid">
       <fieldset className="scheduler-border">
         <legend className="scheduler-border">Upload Addon Package:</legend>
-        <div className="col-sm-9">
-          <input 
-            name="file"
-            type="file" 
-            id="fileInput"
-            accept=".zip"
-            data-text="Browse Addon"
-            data-buttonName="btn-primary"
-            data-placeholder="Select Addon to upload"
-            data-btnClass="btn-primary"
-            className="filestyle" />
-        </div>
-        <div className="btn-toolbar">
-          <button
-            className="btn btn-primary"
-            onClick={handleUpload}>
-            <span 
-              className="glyphicon glyphicon-upload" /> Upload</button>
-          <button
-            className="btn btn-danger"
-            onClick={handleClear}>
-            <span 
-              className="glyphicon glyphicon-remove" /> Clear Addon</button>
+        <div className="row">
+          <div className="col-xs-12">
+            <div className="col-xs-6 col-sm-8 col-md-8 col-lg-8">
+              <input 
+                name="file"
+                type="file" 
+                id="fileInput"
+                accept=".zip"
```

Comment:
Does addons come only as `.zip` files?

Reply:
Yes @kingisaac95, that's the only extension the addon-manager supports now, you can create a ticket to look into supporting other compressed files extension &#x1F60A;